content/index.partial

```diff
@@ -15,6 +15,24 @@
   </div>
 </div>
 
+<!-- Start 5ci -->
+<div id="fiveci" class="row flex">
+  <div class="col-sm-12 col-md-12 col-lg-12">
+    <h1>Getting Started with YubiKey 5Ci</h1>
+    <h3>Yubico SDK for iOS and Android</h3>
+    <h3>Are you developing an iOS or Android app and want to integrate strong authentication support? Now you can with
```

Comment:
Change "integrate strong authentication support" to "integrate strong authentication"